server/bin/www

```diff
@@ -25,10 +24,27 @@ var port = normalizePort(config.floodServerPort);
 app.set('port', port);
 
 /**
- * Create HTTP server.
+ * Create HTTP or HTTPS server.
  */
-
-var server = http.createServer(app);
+ 
+var ssl = Boolean(config.ssl);
```

Comment:
Let's just assume that `config.ssl` is a boolean and not convert the type here. After that, let's just check `if (config.ssl) ` on line 33 instead of creating this variable.